Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -434,7 +434,7 @@ class MessageDetailViewController: OWSViewController, UIScrollViewDelegate {
 
         let contentType = attachment.contentType
         if let dataUTI = MIMETypeUtil.utiType(forMIMEType: contentType) {
-            let attachment = SignalAttachment(dataSource: dataSource, dataUTI: dataUTI)
+            let attachment = SignalAttachment.attachment(dataSource: dataSource, dataUTI: dataUTI)
```

Comment:
The stricter rules around frameworks has exposed that we were using the wrong API here.